modules/boards/spec/features/action_boards/status_board_spec.rb

```diff
@@ -178,6 +178,7 @@
       filters.open
 
       # Expect that status is not available for global filter selection
+      filters.open_available_filter_list
       filters.expect_available_filter 'Status', present: false
```

Comment:
The selector for `expect_available_filter` was wrong, causing it to never find an element, and thus this check always returned true.